prov/efa/configure.m4

```diff
@@ -113,6 +115,44 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_WITH([cmocka],
+		[AS_HELP_STRING([--with-cmocka=DIR],
+				[Provide path to where the Cmocka runtime libraries
+				 are installed.])],
+		[efa_unit_test=yes], [efa_unit_test=no])
+
+	if [test x"$efa_unit_test" = x"yes"] && [test x"$with_cmocka" != x"yes"]; then
+		FI_CHECK_PACKAGE(cmocka,
+						 [cmocka.h],
+						 [cmocka],
+						 [_expect_any],
+						 [],
+						 [$with_cmocka],
+						 [$with_cmocka/lib64],
```

Comment:
I do not think you need this, FI_CHECK_PACKAGE should find the correct libdir for you.

Reply:
done